src/android/TP6mAggregate.jsx

```diff
@@ -15,27 +17,108 @@ import { getData } from '../vendor/perfherder';
 import { withErrorBoundary } from '../vendor/errors';
 import jx from '../vendor/jx/expressions';
 import { Cube, HyperCube, window } from '../vendor/jx/cubes';
-import { g5Reference, TARGET_NAME } from './config';
+import { TARGET_NAME } from './config';
 import ChartJSWrapper from '../vendor/components/chartJs/ChartJsWrapper';
 import timer from '../vendor/timer';
 import { DetailsIcon } from '../utils/icons';
+import { TimeDomain } from '../vendor/jx/domains';
+
+
+const DESIRED_TESTS = ['cold-loadtime'];
+const DESIRED_PLATFORMS = ['p2-aarch64', 'g5'];
+const DESIRED_BROWSER = ['fenix'];
+const REFERENCE_BROWSER = ['fennec64'];
 
 /*
 condition - json expression to pull perfherder data
  */
 async function pullAggregate({
   condition,
-  tests,
-  sites,
-  platforms,
+  test,
+  platform,
   timeDomain,
 }) {
+  const tests = selectFrom(TP6_TESTS).where({ test });
+  const testMode = tests.select('mode').first();
+  const sites = TP6M_SITES.filter(({ mode }) => mode.includes(testMode)).materialize();
+  const platforms = selectFrom(BROWSER_PLATFORMS).where({ platform, browser: DESIRED_BROWSER });
   const readData = timer('read data');
-  const sources = await getData(condition);
+  const referenceRange = new TimeDomain({ past: 'month', ending: 'today' });
+  const referencePlatforms = selectFrom(BROWSER_PLATFORMS).where({ platform, browser: REFERENCE_BROWSER });
+  const referenceFilter = TP6_COMBOS
+    .where({
+      browser: REFERENCE_BROWSER,
+      platform,
+      test,
+    })
+    .select('filter')
+    .toArray();
+  const [fennec64, sources] = await Promise.all([
+    getData({ or: referenceFilter }),
+    getData(condition),
+  ]);
 
   readData.done();
 
   const processData = timer('process data');
+
+  const reference = selectFrom(fennec64)
+    .select('data')
+    .flatten()
+  /* eslint-disable-next-line camelcase */
+    .filter(({ push_timestamp }) => referenceRange.includes(push_timestamp))
+    .map(row => ({ ...row, ...row.meta }))
+    .edges([
+      {
+        name: 'test',
+        domain: {
+          type: 'set',
+          partitions: tests.select({
+            name: 'label',
+            value: 'test',
+            where: 'testFilter',
+          }),
+        },
+      },
+      {
+        name: 'site',
+        domain: {
+          type: 'set',
+          partitions: sites.select({
+            name: 'site',
+            value: 'site',
+            where: 'siteFilter',
+          }),
+        },
+      },
+      {
+        name: 'platform',
+        domain: {
+          type: 'set',
+          partitions: referencePlatforms.select({
+            name: 'platform',
+            value: 'platform',
+            where: 'platformFilter',
+          }),
+        },
+      },
+    ]);
+  const refMax = window(
+    { reference },
+    {
+      edges: ['test', 'site', 'platform'],
+      value: ({ reference }) => selectFrom(reference).select('value').max(),
+    },
+  );
```

Comment:
These `window` functions are not well documented yet:  They operate on multidimensional pivot tables (aka cubes), each dimension/edge of the cube is given a name, the `edges` are the names we will iterate over; in this case over every test, site, platform combination; which which is all the dimensions.  This leaves {reference}, which is an array of data from perfherder, and we take the max of that.  

I suggest you put a debug stop point to better see what's happening in the `value` function. But first, since browser debuggers have been very terrible these past several years, you must expand the `value` function enough to add a breakpoint:

```
value: ({ reference }) => {
    const temp = selectFrom(reference).select('value').max();
    return temp;
},
```